### PR TITLE
NEW: 11ty Template

### PR DESCRIPTION
**Pull request by FredKSchott**
Screen Shot 2020-05-10 at 11 34 10 AM

Screen Shot 2020-05-10 at 11 38 02 AM

### CODE REVIEW

File: templates/app-template-11ty/_includes/layouts/base.njk [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/static/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Web site created using create-snowpack-app" />
+    <link rel="stylesheet" type="text/css" href="/static/index.css" />
+    <title>Snowpack App</title>
+  </head>
+  <body>  
+    <noscript>You need to enable JavaScrifddpt to run this app.</noscript>
```
Comment (JakeChampion) [inline templates/app-template-11ty/_includes/layouts/base.njk:12]:
```suggestion
    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment (nickcolley) [reply templates/app-template-11ty/_includes/layouts/base.njk:12]: Given that Eleventy is pretty pro server-first should the template even have this in it? Most Eleventy sites work without JavaScript.

Comment (FredKSchott) [reply templates/app-template-11ty/_includes/layouts/base.njk:12]: I can't stop thinking about how `JavaScrifddpt` got in there...

Comment (FredKSchott) [reply templates/app-template-11ty/_includes/layouts/base.njk:12]: this response on Twitter answers it well: https://twitter.com/FredKSchott/status/1259596071891636225

basically, JS is the main thing that you'd use this template for. But, you're obviously welcome to remove this in your app.

**Comment by zachleat** [comment]
Ah hmm. if that's a requirement it does kind of go against everything Eleventy was made for, unfortunately. I'd hate for people to start an Eleventy project with this and be blindsided with this core architectural difference. So much so that I would request this not to be merged 😭

Is there any way to make it work in a No-JS way?

**Comment by FredKSchott** [comment]
100%! To be clear, it was a requirement of the template as written and not of Snowpack. If this is really important to the 11ty project philosophically, I'd love to make a starter template that works with that requirement.

As a general rule we're keeping all CSA templates pretty simple, so that we don't get into this position of pushing for/against conventions.

At the end of the day Snowpack is a dev environment for web apps, just like 11ty. It's as dependent/JS-required as whatever you build with it.

**Comment by FredKSchott** [comment]
Warning removed!

**Comment by FredKSchott** [comment]
Merging assuming this is good (concern addressed).

**Pull request merged**